maestrowf/__init__.py

```diff
@@ -41,6 +41,8 @@
 import logging
 import sys
 
+from importlib_metadata import version  # Replace upon deprecation of python 3.7
```

Comment:
Happen to know what this gets replaced with once 3.7 is deprecated? Is there an existing solution now?

Reply:
Oh scratch that -- this is the backport. I see. I know it's a bit uglier but, mind replacing this with the usual
```python
try:
    from importlib.metadata import version
except ImportError as imp_err:
    from importlib_metadata import version
```

I think it makes it clearer that there's a backport and then when 3.7 is EOL'ed we can just remove the except.

Reply:
Done.